org.tlauncher.TLauncher.metainfo.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 TLauncher Inc. -->
+<component type="desktop-application">
+  <id>org.tlauncher.TLauncher</id>
+  <name>TLauncher</name>
+  <summary>The launcher for Minecraft</summary>
+  <developer_name>TLauncher Inc.</developer_name>
+
+  <description>
+    <p>Today not many good launcher for the game Minecraft, but they are
+    still there, and one of them TLauncher.</p>
+
+    <p><em>All official versions, from Alpha to Snapshots:</em>The launcher
```

Comment:
Currently anything within `<em>` tags ends up as `(null)` on the website. You may want to remove it.

Reply:
Ah well.